doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,297 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+
+- The tasks, actors, and placement groups broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
+
+.. _dash-workflow-resource-utilization:
+
+Comparing the hardware and logical resource utilization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ray requires users to specify the number of :ref:`resources <logical-resources>` their tasks and actors will use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``. 
+These values are used for scheduling, but may not always match the actual resource utilization. 
+
+If the logical resources don't match the physical resource usage, it can lead to the under-utilization of the cluster because the number of tasks and actors are throttled by
+the logical resource requirements.
+
+You can check the :ref:`Metrics View <dash-metrics-view>` to compare logical and hardware resource utilization. 
+
+Let's see an example.
+
+.. code-block:: python
+
+    @ray.remote(num_cpus=1)
+    def task():
+        import time
+        time.sleep(30)
+    
+    # Launch 30 tasks
+    ray.get([task.remote() for _ in range(30)])
+
+When you run the below code that executes many tasks that sleep, 
+you will see from the metrics page that there are many logical CPU allocations but little to no hardware CPU usage.
+It is because the logical resource requirement is 1 CPU for each task, but each task uses nearly no CPU because it just sleeps.
+
+TODO-SANG Add images
+
+resource specification for those tasks will lead to wasted physical resources. To address this issue, let's adjust the ``num_cpus`` value to 0.1 so that other tasks or actors can be scheduled to leverage the hardware resources.
+
+.. code-block:: python
+
+    @ray.remote(num_cpus=0.1)
+    def task():
+        import time
+        time.sleep(30)
+
+.. _dash-workflow-logs:
+
+View the application logs and errors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Driver Logs**
+If the Ray job is submitted by :ref:`Ray job API <jobs-quickstart>`, the job logs are available from the dashboard. The log file follows the following format; ``job-driver-<job_submission_id>.log``.
+
+.. note:: 
+
+  If the driver is executed directly at the Ray cluster (without the job API), the driver logs are not accessible from the dashboard. In this case, see the terminal output to view the driver logs.
+
+TODO-SANG Show the log button at the job row.
+
+**Task and Actor Logs**
+Task and actor logs are accessible from the :ref:`task and actor table view <dash-workflow-state-apis>`. Click the log button.
+You can see the worker logs that execute the task and actor. ``.out`` (stdout) and ``.err`` (stderr) logs contain the logs emitted from the tasks and actors. The core worker logs contains the system-level logs for the corresponding worker.
```

Comment:
"You can see the worker logs (`worker-...`) that execute ..."

"Usually, you don't need to look at core worker log files (`python-core-worker-...`) since they store system-level logs."

How about these two changes?